aibolit/patterns/method_chaining/method_chaining.py

```diff
@@ -0,0 +1,41 @@
+import javalang
+
+from aibolit.utils.utils import remove_comments
+
+
+class MethodChainFinder:
+
+    def __init__(self):
+        pass
+
+    def __file_to_ast(self, filename: str) -> javalang.ast.Node:
+        """
+        Takes path to java class file and returns AST Tree
+        :param filename:
+        :return: Tree
+        """
+        with open(filename, encoding='utf-8') as file:
+            res = javalang.parse.parse(remove_comments(file.read()))
+        return res
+
+    # flake8: noqa: C901
+    def value(self, filename: str):
```

Comment:
@lyriccoder I coudn't figure out what this method is returning. Please add some documentation to it